PWGDQ/Tasks/tableReader.cxx

```diff
@@ -351,6 +363,196 @@ struct AnalysisMuonSelection {
   PROCESS_SWITCH(AnalysisMuonSelection, processDummy, "Dummy function", false);
 };
 
+struct AnalysisPrefilterSelection {
+  Produces<aod::Prefilter> prefilter;
+  Preslice<MyBarrelTracks> perCollision = aod::reducedtrack::reducedeventId;
+
+  // Configurables
+  Configurable<std::string> fConfigDalitzTrackCutsId{"cfgDalitzTrackCutsId", "", "Id of Dalitz track selection cuts (from cfgTrackCuts in AnalysisTrackSelection, starting at 0 = first dalitz cut)"};
+  Configurable<std::string> fConfigDalitzPairCuts{"cfgDalitzPairCuts", "", "Dalitz pair selection cuts"};
+  Configurable<std::string> fConfigAddTrackPairHistogram{"cfgAddTrackPairHistogram", "", "Comma separated list of histograms"};
+  Configurable<bool> fQA{"cfgQA", false, "QA histograms"};
+
+  Filter filterBarrelTrackSelected = aod::dqanalysisflags::isBarrelSelectedDalitz > 0; // Dalitz candidates
+
+  OutputObj<THashList> fOutputList{"output"}; //! the histogram manager output list
+  OutputObj<TList> fStatsList{"Statistics"};  //! skimming statistics
+
+  std::map<int, uint8_t> dalitzmap;
```

Comment:
We should respect some conventions for the naming of variables, for easier code readability. Usually, the class data members start with an "f" followed by a capital. So in this case, "dalitzmap" -> "fDalitzMap". But find a better name, since Dalitz should be removed.